model/manifest.go

```diff
@@ -108,6 +111,7 @@ type PluginSettingsSchema struct {
 //      "description": "This is my plugin",
 //      "homepage_url": "https://example.com",
 //      "support_url": "https://example.com/support",
+//		"release_notes_url": "https://example.com/releases/v0.0.1",
```

Comment:
I forgot to add this in https://github.com/mattermost/mattermost-server/pull/13677

Reply:
Indentation looks off here?

Reply:
Good catch :+1: Fixed